app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = (sqlite3.connect
+                            ("C:/Users/bilou/projects/"
+                             "py-actor-manager/cinema.sqlite"))
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in cursor
+        ]
+
+    def update(self, id: int, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name = ?, last_name = ? "
+            f"WHERE id = ? ",
+            (id, first_name, last_name)

```

Comment:
The parameters in the `execute` method for the `update` function are incorrectly ordered. The `id` should be the last parameter in the SQL query, not the first. Change the order of parameters to `(first_name, last_name, id)` to match the SQL query.